Example/Podfile

```diff
@@ -15,7 +15,7 @@ target 'Emission' do
   pod 'Emission', :path => '../'
 
   # As this runs dev, we need the developer web socket
-  pod 'React', :path => react_path, :subspecs => %w(RCTWebSocket)
+  pod 'React', :path => react_path, :subspecs => %w(DevSupport BatchedBridge RCTAnimation)
```

Comment:
The `RCTAnimation` subspec causes problems, but I can build Emission [if this postinstall script is included in package.json](https://github.com/facebook/react-native/issues/13198#issuecomment-302917321).